CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Added ability to set a custom config directory via the new
   `APPTAINER_CONFIGDIR` environment variable.
 
+### Other changes
+
+- Change `--underlay` help message.
+
```

Comment:
Let's leave out a CHANGELOG entry for this, it is too trivial (and anyway it would need to be in pre-release section if we had one).

Reply:
okay, removed in this commit
https://github.com/apptainer/apptainer/pull/1473/commits/7e00a0819b3c2babb2df5185bd3430f00ddb97df